services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -430,7 +430,7 @@ public RouteTableWebJson updateSubnetRouteTable(@PathVariable String projectid,
             }
             newRoutes.setRoutes(routes);
 
-            UpdateRoutingRuleResponse updateRoutingRuleResponse = this.neutronRouterService.updateRoutingRule(subnetid, newRoutes, false);
+            UpdateRoutingRuleResponse updateRoutingRuleResponse = this.neutronRouterService.updateRoutingRule(subnetid, newRoutes, false, false);
```

Comment:
@kevin-zhonghao How do you know it's *delete* operation here?

Reply:
Here should configure as True